aws_topology/stackstate_checks/aws_topology/aws_topology.py

```diff
@@ -65,8 +62,19 @@ def get_account_id(self, instance_info):
     def get_instance_key(self, instance_info):
         return TopologyInstance(self.INSTANCE_TYPE, str(self.get_account_id(instance_info)))
 
-    def must_run_full(self, instance_info, interval):
-        return seconds_ago(instance_info.state.last_full_topology) > interval
+    def must_run_full(self, interval):
+        self.log.info('Checking if full run is necessary')
+        if not hasattr(self, 'last_full_topology'):
+            # Create empty state
+            self.log.info('  Result => YES (first run)')
+            return True
+        secs = seconds_ago(self.last_full_topology)
+        self.log.info('  Result => {} (Last run was {} seconds ago, interval is set to {})'.format(
+            "YES" if secs > interval else "NO",
+            int(secs),
+            interval)
+        )
+        return secs > interval
```

Comment:
Pieter reported that the topology could be running every 61 minutes instead of 60 minutes, this could be the issue as it's expecting it to be more than the interval

Reply:
I think this fixes this